src/css/settings/_transitions.scss

```diff
@@ -24,3 +24,27 @@ $transition: $transition-duration-short $transition-easing;
 %transition-long {
   transition: all $transition-duration-long $transition-easing;
 }
+
+@keyframes animation-fade-in-up-slow {
+  35% {
+    opacity: 0;
+    transform: translate3d(0, -1rem, 0);
+  }
+
+  100% {
+    opacity: 1;
+    transform: translate3d(0, 0, 0);
+  }
+}
+
+@keyframes animation-fade-in-up-slow-2 {
+  0% {
+    opacity: 1;
+    transform: translate3d(0, 0, 0);
+  }
+
+  50% {
+    opacity: 0;
+    transform: translate3d(0, -1rem, 0);
+  }
+}
```

Comment:
maybe these should be animation-fade-in-down and animation-fade-out-up?

Reply:
Changed!

Parcel bundler requires .html but when the site is launched we will remove these!